### DISCUSSION TITLE
ProArt theme from turzx.

### DISCUSSION THREAD

**Discussion by Helodix** [mathoudebine/turing-smart-screen-python#501] [Q&A] [answered]
Hello guys, do you have any ideas how I could recreate this theme? I'm newbie at coding :/

http://discuz.turzx.com/d/564-35cun-35inch-zi-zhi-hua-shuo-proartxi-lie-zhu-ti-fen-xiang-asus-proart-theme-shring

**Comment by c0ldJS** [comment, marked as answer]
Readers: No HTTPS on provided link so browse accordingly.

@Helodix : Here's a loose rundown of one possible route to take:

- Download the .ZIP provided there to obtain the background image (so you don't have to recreate it)
- If it isn't already, get this project's application up and running on your display
- In `~/res/themes/`, make a new directory with the name you'd like the theme to have
- Copy the background image to this new directory
- Copy `~/res/themes/theme_example.yaml` to the new directory, and rename it to `theme.yaml`
- `py theme-editor.py [NAME OF THEME]` will open the file you need to edit and a virtualized display so you can see your changes take effect (it'll update when you save changes to the theme file)
- Start playing

The initial theme file you'll be starting with has all of the stuff built for the metrics used in the final layout, so it'll be an exercise in activating the ones you'd enjoy using and getting the colors and placement squared away. The YAML is well documented and the pre-filled content can be used as an additional guide on how to do things. Don't forget that there's also a [wiki section devoted to explaining theme creation](https://github.com/mathoudebine/turing-smart-screen-python/wiki/System-monitor-%3A-themes). No real coding is necessary; a lot of the work will be the X and Y coordinates for where things need to go and sizing the typefaces properly.

The radial graphs are probably the most complex thing to assemble in the file. Most everything else should be pretty self-explanatory.

It does mean some trial and error but the tradeoff is being able to take that theme wherever you want it to go. Wish the CPU utilization was an animated line graph over time instead of a percentage? You can totally do that. Use different typefaces. Change the color so that it better matches the ProArt hardware, or give each metric its own dedicated color.

**Reply by Helodix** [reply]
Luckily I own the background image but not sure how to do the radial graphs. Can’t find command/script for that in tutorial. Thanks for help!

**Reply by Helodix** [reply]
Okay, after you comment I figured out everything. Much thanks!
